R/pkg/R/mllib.R

```diff
@@ -602,14 +599,14 @@ setMethod("spark.survreg", signature(data = "SparkDataFrame", formula = "formula
 # Returns a summary of the AFT survival regression model produced by spark.survreg,
 # similarly to R's summary().
 
-#' @param object A fitted AFT survival regression model
+#' @param object a fitted AFT survival regression model.
 #' @return \code{summary} returns a list containing the model's coefficients,
 #' intercept and log(scale)
 #' @rdname spark.survreg
 #' @export
 #' @note summary(AFTSurvivalRegressionModel) since 2.0.0
 setMethod("summary", signature(object = "AFTSurvivalRegressionModel"),
-          function(object, ...) {
+          function(object) {
```

Comment:
thinking more about, I think the reason is because R's base::summary has in fact the `...` https://stat.ethz.ch/R-manual/R-devel/library/base/html/summary.html

Could you test if base::summary still work? I think we would prefer omitting `...` for all our `summary` functions but we need to make sure doing so doesn't mask it